### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by zishanmirza**
This patch updates the validation for the principal investigator.

- [x] Tests passing
- [X] Black formatting
- [ - ] Migrations for any changes to the database schema
- [X] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

### CODE REVIEW

**Comment by zishanmirza** [comment]
Fixes https://github.com/ScilifelabDataCentre/dds_cli/issues/406.

File: dds_web/api/schemas/project_schemas.py [hunk 1]
```diff
@@ -88,10 +88,7 @@ class Meta:
     pi = marshmallow.fields.String(
         required=True,
         allow_none=False,
-        validate=marshmallow.validate.And(
-            marshmallow.validate.Length(min=1, max=255),
-            dds_web.utils.contains_disallowed_characters,
-        ),
+        validate=marshmallow.validate.Email(),
```
Comment (inaod568) [inline dds_web/api/schemas/project_schemas.py:91]: You also need to replace the lines here: https://github.com/zishanmirza/dds_web/blob/54eff2d3c739de1d628bb18a0c18c3d26bfa471c/dds_web/api/project.py#L571-L572 

with this: 
```
        # Add a new project to db
        try:
            new_project = project_schemas.CreateProjectSchema().load(p_info)
            db.session.add(new_project)
        except sqlalchemy.exc.OperationalError as err:
            raise DatabaseError(message=str(err), alt_message="Unexpected database error.")
```

**Review by inaod568 — CHANGES_REQUESTED** [review]

File: dds_web/api/schemas/project_schemas.py (hunk 1 above)
Comment (zishanmirza) [reply dds_web/api/schemas/project_schemas.py:91]: Sure, I can add a "try" statement.

**Comment by inaod568** [comment]
@zishanmirza Can you also add row in the change log?

**Review by inaod568 — CHANGES_REQUESTED** [review]
Add the changelog and then I'm ready to approve 👍🏻

**Comment by zishanmirza** [comment]
I have updated CHANGELOG.md.

**Comment by zishanmirza** [comment]
I have updated CHANGELOG.md.

**Comment by zishanmirza** [comment]
I have updated CHANGELOG.md.

**Review by inaod568 — APPROVED** [review]

**Review by inaod568 — APPROVED** [review]

**Pull request merged**